.gitattributes

```diff
@@ -0,0 +1,13 @@
+.* export-ignore
+*.md export-ignore
+tests export-ignore
+docs export-ignore
+examples export-ignore
+mongo-orchestration export-ignore
+tools export-ignore
```

Comment:
I think it makes sense to include `examples/` and `tests/` in exports since they're intended for our users. @alcaeus: thoughts?

Reply:
I would prefer to refer to the documentation in this case. If someone really needs this, one can hop over to github or use prefer source composer option.

We save a lot of traffic when thinking about CI ✌🏻

It's up to you 🤓

Reply:
I'm admittedly in the minority of users that frequently jumps through the `vendor/` directory to reference code. The impact on CI and users specifying `--prefer-source` are both good points. I'm on board with stripping all of this from the git archive in that case.

Reply:
Thank you 👍